kola/tests/misc/tpm.go

```diff
@@ -25,14 +25,13 @@ const (
 			"version": "3.3.0"
 		},
 		"kernelArguments": {
-			"shouldExist": ["rd.luks.name=12345678-9abc-def0-1234-56789abcdef0=rootencrypted", "systemd.mask=systemd-cryptsetup@rootencrypted.service"]
+			"shouldExist": ["systemd.mask=systemd-cryptsetup@rootencrypted.service"]
```

Comment:
With this removed the whole kernelArguments section wouldn't be needed and we can skip the reboot that Ignition does.

Reply:
Yup, it's removed now.